include/mbedtls/pk.h

```diff
@@ -234,10 +234,17 @@ typedef struct mbedtls_pk_info_t mbedtls_pk_info_t;
 
 /**
  * \brief           Public key container
+ *
+ * \note            The priv_id is guarded by MBEDTLS_PSA_CRYPTO_C and not
```

Comment:
Note: you should be able to remove this comment when addressing https://github.com/Mbed-TLS/mbedtls/issues/7126#issuecomment-1535972352 (no change needed now, just noting for reference).

Reply:
Yes, I think so. Indeed I added the comment there because I thought it might look weird for a reviewer to see a different guard.
Following the discussion we had on slack, I think that it should be better to create a new issue for this activity on `mbedtls_pk_sign_ext` instead of using #7126 so that we can also mention this last comment there and have everything updated correctly at once (my memory is not that good to remember all the details :D ). I'll do that later today

Reply:
Yes, please feel free to split 7126 into two issues. Don't forget to add something about checking if `build_info.h` can be adjusted to each issue!

Reply:
The comment is also hard to understand, since it seems to me as if the priv_id should be guarded by both of these defines (after reading the comment). I won't complain since I can see that it's about to change.

Reply:
Do you think that removing the `only` word helps in making it more clear?
```suggestion
 * \note            The priv_id is guarded by MBEDTLS_PSA_CRYPTO_C and not
 *                  by MBEDTLS_USE_PSA_CRYPTO because it can be used also
 *                  in mbedtls_pk_sign_ext for RSA keys.
```

Reply:
Yes! That's better, thanks! :)